.github/workflows/documentation.yml

```diff
@@ -5,24 +5,12 @@
 # * checkout PR head commit https://github.com/actions/checkout#checkout-pull-request-head-commit-instead-of-merge-commit
 name: documentation
 on:
+  pull_request:
   push:
-    branches:
-      - master
+    branches: [master]
```

Comment:
we were even already doing this here! I changed the formatting to be identical across all of the workflows; I think this syntax is a bit nicer than the previous one.